mappings/net/minecraft/sortme/BooleanMerger.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS ckc net/minecraft/sortme/BooleanMerger
+	FIELD AND AND Lckc;
```

Comment:
no not those fields

enigma will eventually autodetect them

Reply:
Hmm, they are marked as obfuscated

Reply:
>enigma will eventually autodetect them